docker/statutory/bootstrap.sh

```diff
@@ -3,3 +3,5 @@ echo "Creating database..."
 npm run db:create
 echo "Migrating database..."
 npm run db:migrate
+
+sh -c "${@}"
```

Comment:
this one isn't needed if we have 2 commands

Reply:
Ah yes